server/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12.18.1
+WORKDIR /app
+ENV NODE_ENV=production
+COPY ["package.json", "package-lock.json*", "./"]
+RUN npm install --production
+COPY . .
```

Comment:
You should add a .dockerignore to prevente copy of non production files

Reply:
I'm no NodeJS expert thus I can't tell which files to omit. Nevertheless from my limited knowledge I couldn't find any of the files in the server folder that could be added to `.dockerignore` at the moment. Or do you suggest adding an empty `.dockerignore` file?

Reply:
I've added a .dockerignore file. Would you mind having a look at it and suggest changes if needed? Thanks!